ui/Screen/Project/ProjectHeader.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="typescript">
+  import type { Stats } from "../../src/project";
+
+  import { Icon } from "../../DesignSystem/Primitive";
+  import { RadicleId } from "../../DesignSystem/Component";
+
+  export let name: string;
+  export let urn: string;
+  export let description: string = "";
+  export let stats: Stats | undefined = undefined;
+  export let onClick: (() => void) | undefined = undefined;
+</script>
+
+<style>
+  .metadata {
+    display: flex;
+    flex-direction: column;
+    align-self: center;
+    width: -webkit-fill-available;
+    min-width: 0;
+  }
+
+  .description {
+    margin-top: 1rem;
+    margin-bottom: 0.5rem;
+    color: var(--color-foreground-level-6);
+  }
+
+  .stats {
+    display: flex;
+    margin-top: 0.5rem;
+  }
+
+  .stat-item {
+    display: flex;
+    color: var(--color-foreground-level-6);
+    margin-right: 1rem;
+  }
+
+  .stat-item p {
+    margin-left: 0.5rem;
+    white-space: nowrap;
+  }
+
+  .stat-separator {
+    display: flex;
+    color: var(--color-foreground-level-3);
+    margin-right: 1rem;
+  }
+
+  .clickable {
+    cursor: pointer;
+  }
+</style>
+
+<div class="metadata">
+  <h1
+    data-cy="entity-name"
+    class="typo-overflow-ellipsis"
+    title={name}
+    class:clickable={onClick !== undefined}
+    on:click={() => (onClick ? onClick() : {})}>
+    {name}
+  </h1>
+  <RadicleId truncate {urn} style="margin-top: 0.5rem;" />
+  {#if description.length > 0}
+    <p class="description typo-overflow-ellipsis" title={description}>
+      {description}
+    </p>
+  {/if}
+  {#if stats}
+    <div class="stats" data-cy="project-stats">
+      {#if stats && stats.branches > 0}
```

Comment:
Can you check if we need to check for `stats` or if the following is enough?
```suggestion
      {#if stats.branches > 0}
```
Same also goes for the two if-statements below

Reply:
c6987c12f30e2263784da02d39c706d59bda3ff9